src/DotNetWorker.Core/StartupHook.cs

```diff
@@ -35,7 +35,8 @@ public static void Initialize()
 #if NET5_0_OR_GREATER
         int processId = Environment.ProcessId;
 #else
-        int processId = Process.GetCurrentProcess().Id;
+        using var process = Process.GetCurrentProcess();
```

Comment:
Does this code compile? "Using declarations" is available from C# 8 onwards. For a .net framework function app that is not the case.

Reply:
c# version  has nothing to do with target framework. eg u can use c# 12 to target .net 472